packages/components/CHANGELOG.md

```diff
@@ -46,7 +49,6 @@
 
 -   `ColorPalette`, `BorderControl`: Don't hyphenate hex value in `aria-label` ([#52932](https://github.com/WordPress/gutenberg/pull/52932)).
 -   `MenuItemsChoice`, `MenuItem`: Support a `disabled` prop on a menu item ([#52737](https://github.com/WordPress/gutenberg/pull/52737)).
--   `TabPanel`: Introduce a new version of `TabPanel` with updated internals and improved adherence to ARIA guidance on `tabpanel` focus behavior while maintaining the same functionality and API surface.([#52133](https://github.com/WordPress/gutenberg/pull/52133)).
```

Comment:
@torounit Just noticed that this PR removed a CHANGELOG entry — was that on purpose? If not, could you add it back (maybe as part of #51350) ?

Reply:
This is a mistake. Commit to PR #51350.